### DISCUSSION TITLE
Python API stubs for type hints and auto-completion

### DISCUSSION THREAD

**Discussion by mathiaswold** [carla-simulator/carla#5755] [Show and tell]
Hello!

Issue https://github.com/carla-simulator/carla/issues/1333 was closed with no solution as far as I can tell.

I have created a repository containing Python stubs for the CARLA Python API. Installation instructions can be found there: https://github.com/mathiaswold/carla-python-stubs.

Installing the Python stubs will allow you to use type hints and auto-completion in your code editor. Hope this is useful!

**Comment by RaviBeagle** [comment]
@mathiaswold, Thanks a lot. I tried this and its working good.

I was thinking of solving the problem by installing the Python package itself. 
The installation documentation  https://carla.readthedocs.io/en/latest/start_quickstart/
It provides the whl file and Downloadable Python package methods too. But I am not sure if this would solve the problem.
